github/ci/prow/files/jobs/virtblocks/virtblocks/virtblocks-periodics.yaml

```diff
@@ -5,13 +5,39 @@ presubmits:
     always_run: true
     spec:
       containers:
-        - image: kubevirt/golang-libtool:1.12.5-r1
+        - image: quay.io/virblocks/buildenv:1.0.0
```

Comment:
should not the name be `virtblocks`?

Reply:
> should not the name be `virtblocks`?

It definitely should, good catch! I'll fix it right away.

Reply:
Done!